config/routes.rb

```diff
@@ -2755,9 +2755,9 @@
   API_VERSION_REGEX = /v[\d]+(\.[\da-zA-Z]+)*(\-[\da-zA-Z]+)?/ unless defined?(API_VERSION_REGEX)
 
   # OPTIONS requests for REST API pre-flight checks
-  match '/api/*path' => 'manage_i_q/a_p_i/base#handle_options_request', :via => [:options]
+  match '/api/*path' => 'a_p_i/base#handle_options_request', :via => [:options]
 
-  get '/api(/:version)' => 'manage_i_q/a_p_i/base#show_entrypoint', :format => 'json', :version => API_VERSION_REGEX
+  get '/api(/:version)' => 'a_p_i/base#show_entrypoint', :format => 'json', :version => API_VERSION_REGEX
```

Comment:
@Fryguy if I use `'api/base#show_entrypoint'` etc.. for these, I get this, across the board:

```
  1) Events API Event collection query invalid event
     Failure/Error: get url, :params => options.stringify_keys, :headers => update_headers(headers)
     
     LoadError:
       Unable to autoload constant Api::BaseController, expected /home/tim/src/manageiq/app/controllers/api/base_controller.rb to define it
     # ./spec/support/api_spec_helper.rb:33:in `run_get'
     # ./spec/requests/api/events_spec.rb:21:in `block (3 levels) in <top (required)>'
```